### PR TITLE
Update node management section in run-blockchain/index

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #[745](https://github.com/LiskHQ/lisk-core/issues/745) in lisk-core

### CODE REVIEW

**Review by chris529 — APPROVED** [review]
LGTM! 👍

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 1]
```diff
@@ -62,10 +63,14 @@ The "Run Blockchain" section contains all the guides concerning the management o
...
 A blockchain node is a server with a running blockchain client, such as xref:{url_core}[].
-Nodes are usually connected to other nodes running the same client via P2P.
-// TODO: (See above comment on P2P) - Add in P2P link {url_p2p}[P2P^].
-Together, they form the network that is maintaining the blockchain.
+Nodes are usually connected to other nodes which are running an instance of the same client via P2P.
+Node operators are required to set up a node on a server, and then connect it to the desired network.
+
+There are almost 2,000 nodes around the world that are maintained by individuals, and these nodes communicate with the network.
+Together, they form the network that is maintaining the Lisk blockchain.
+For example, by broadcasting and receiving blocks or transactions from their peers.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]: Would remove this part or move to Lisk Core docs under https://liskhq.github.io/lisk-docs/lisk-core/v4/index.html#networks 

Also, I am not sure if the number 2,000 is up to date, we should probably double-check it is roughly correct, or remove the number.

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 2]
```diff
@@ -193,9 +198,32 @@ Once configured, a relayer node can directly connect with the RPC nodes of the r
 
 == Node management
 
+=== Who should operate a node?
+
+If you fall under one of the following categories listed below, then it is recommended to set up your own node:
+
+* *Exchanges* and other services that rely on a stable API interface to the network.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]: Add here:

- 
```suggestion
* *Exchanges, Web3 app developer*and other services that rely on a stable API interface to the network.
* *Relayers*: Who participate in the Lisk cross-chain communication by relaying CCUs.
```

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 3]
```diff
@@ -193,9 +198,32 @@ Once configured, a relayer node can directly connect with the RPC nodes of the r
...
+* *Validators* who have registered as a validator and would like to actively generate new blocks.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]: To become validator, you need to register, so the statement was a little redundant.

```suggestion
* *Validators* who would like to actively generate new blocks.
```

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 4]
```diff
@@ -193,9 +198,32 @@ Once configured, a relayer node can directly connect with the RPC nodes of the r
...
+* *Users* who do not trust external sources and want to be in full control over their node.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]:
```suggestion
* *Users* who do not trust external sources and want to be in full control over their own node.
```

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 5]
```diff
@@ -193,9 +198,32 @@ Once configured, a relayer node can directly connect with the RPC nodes of the r
...
+
+- To have a private entry point to communicate with the network.
+This is especially important when running an exchange and implementing LSK tokens.
+- To create your own snapshots of the blockchain.
+- To create transactions and send them to the network.
+//TODO: Add link back, once docs are updated
+//- To have the option to xref:{url_config_forging}[generate] new blocks, (assuming you are an active delegate).
+//- To acquire full control in order to xref:{url_config}[configure] the node to your specific requirements.
+- To have the option to generate new blocks, (assuming you are an active validator).
+- To acquire full control in order to configure the node to your specific requirements.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]: This sections sounds redundant to the previous section. Would remove/merge the two sections.

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 6]
```diff
@@ -193,9 +198,32 @@ Once configured, a relayer node can directly connect with the RPC nodes of the r
...
+
 Related guides::
+** xref:{url_configuration}[Client Configuration]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]: Would add this link under `How to set up a Lisk Mainnet node` section, under the setup guides, or remove here.

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 7]
```diff
@@ -193,9 +198,32 @@ Once configured, a relayer node can directly connect with the RPC nodes of the r
...
+** xref:{url_run_pm2}[]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]: Would add this link under `How to set up a Lisk Mainnet node` section

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/run-blockchain/index.adoc (hunk 1 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/run-blockchain/index.adoc]: Text removed

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**